src/components/PlantsV2/components/HighestAndLowestMortalityRateSpeciesCard.tsx

```diff
@@ -20,19 +20,19 @@ export default function HighestAndLowestMortalityRateSpeciesCard({
   let highestSpecies = '';
 
   observation?.species.forEach((sp) => {
-    if (sp.mortalityRate && sp.mortalityRate > highestMortalityRate) {
+    if (sp.mortalityRate !== undefined && sp.mortalityRate >= highestMortalityRate) {
```

Comment:
Thanks for the changes @constanzauanini 
Just curious... I see a lot of `null` mortalityRates as well... any idea what we should do about those?
Do we need a !== undefined and !== null check? A bit confused for my own purposes.

Reply:
I think value != null will be true for null or undefined, but I don't think the linter will allow the double equals (instead of triple)

Reply:
sounds good, maybe just 2 separate checks..
here's what I see in staging  for https://staging.terraware.io/observations/56/results/5/zone/107 (dev org)
<img width="388" alt="DevTools - staging terraware io:observations:56:results:5:zone:107 2023-07-06 08-07-50" src="https://github.com/terraware/terraware-web/assets/1865174/61867920-27ce-4937-9341-a11aabc473fb">